downloads/swan-lake-release-notes/swan-lake-beta1/RELEASE_NOTE.md

```diff
@@ -1,46 +1,232 @@
+- Improved the compiler plugin to validate inputs and return types
+- Added support to use included record parameters instead of the `record` type in the listener initialization
+
+#### `http` Package
+- Improved the `http:Client` remote methods to support the contextually-expected type inference
+- Changed the configuration parameters of the listeners and clients to include the record parameters
+- Populated the error detail of `http:ClientRequestError` and `http:RemoteServerError` with response payload and header
+- Swapped the `mediaType` and `headers` arguments in the client remote methods
+
+#### `sql` Package
+- Added the SQL Array Value type support and introduced the new distinct array value types for identified SQL types
+- Marked the `sql:Client` class as an isolated class
+
+#### `websubhub` Package
+Included the auth configuration to the WebSubHub publisher client configuration
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%22Swan+Lake+Beta1%22+label%3AType%2FBug).
+
+## Code to Cloud Updates
+
+### New Features
+Added `@cloud:Expose` annotation to support custom listeners.
+
+### Bug Fixes
+
+To view bug fixes, see the GitHub milestone for Swan Lake Beta1 of the repositories below.
+
+- [C2C](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22)
+- [Docker](https://github.com/ballerina-platform/module-ballerina-docker/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22)
+- [AWS Lambda](https://github.com/ballerina-platform/module-ballerinax-aws.lambda/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22)
+- [Azure Functions](https://github.com/ballerina-platform/module-ballerinax-azure.functions/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22) 
+
+## Developer Tools Updates
+
+### Language Server 
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22+label%3ATeam%2FLanguageServer).
+
+### New Features
+
+#### Debugger
+- Added remote debugging support for the command, which runs the Ballerina executable JAR
+- Added support for debugging single Ballerina tests in Visual Studio Code
+
+### Bug Fixes
+
+To view bug fixes, see the GitHub milestone for Swan Lake Beta1 of the repositories below.
+
+- [Language](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22+label%3ATeam%2FLanguageServer)
+- [OpenAPI](https://github.com/ballerina-platform/ballerina-openapi/milestone/5?closed=1) 
+
+### Ballerina Packages Updates
+
+## Breaking Changes
```

Comment:
Shouldn't we move this topic to somewhere else. Having this here is not appropriate IMO, It should be a topic under one of the  above 4 high level sections.

Reply:
Moved under "Language Breaking Changes".